### PR TITLE
Added doc

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
We need to add the following secrets:
- SPHINX_DOCUMENTATION
- PYPI_API_TOKEN
- PYTEST_COVERAGE_COMMENT (dette kræver også en gist)

I don't seem to be able to do it. Let's discuss it when we get the time.

I have included some sample files in the documentation if you want to fill them out. Also included two logos (dark and light mode)

### CODE REVIEW

File: .github/workflows/publish_to_pypi.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (HLasse) [inline .github/workflows/publish_to_pypi.yml:34]: secret has been added

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ["_static"]
+
+html_show_sourcelink = True
+
+html_context = {
+    "display_github": True,  # Add 'Edit on Github' link instead of 'View page source'
+    "github_user": "KennethEnevoldsen",
```
Comment (HLasse) [inline docs/conf.py:72]:
```suggestion
    "github_user": "HLasse",
```

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+# ones.
+extensions = [
+    "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
+    "sphinxext.opengraph",
+    "sphinx_copybutton",
+    "sphinx.ext.githubpages",
+]
+
+# autodoc_mock_imports = ["dacy"]
```
Comment (HLasse) [inline docs/conf.py:45]:
```suggestion
# autodoc_mock_imports = ["textdescriptives"]
```

File: docs/installation.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Installing from source
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also install DaCy directly from source using:
+
+.. code-block::
+
+   git clone https://github.com/hlasse/textdescriptives.git
+   cd DaCy
```
Comment (HLasse) [inline docs/installation.rst:18]:
```suggestion
   cd TextDescriptives
```

File: docs/installation.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+.. code-block::
+
+   pip install textdescriptives
+
+
+Installing from source
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also install DaCy directly from source using:
```
Comment (HLasse) [inline docs/installation.rst:13]:
```suggestion
You can also install TextDescriptives directly from source using:
```

**Review by HLasse — COMMENTED** [review]
very nice, let's finish it up by adding the remaining secret keys when we meet again

File: docs/conf.py (hunk 2 above)
Comment (KennethEnevoldsen) [reply docs/conf.py]: bliver egentlig ikke brugt fordi vi bruger 'View page source'

**Comment by KennethEnevoldsen** [comment]
Good catches. Let's look at the secrets monday

**Pull request merged**